tests/security/cc/ipsec_client.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run CC 'ipsec' client case
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#101226
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test;
+use Utils::Architectures;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    my $server_ip = get_var('SERVER_IP', '10.0.2.101');
+    my $client_ip = get_var('CLIENT_IP', '10.0.2.102');
+
+    # We don't run setup_multimachine in s390x, but we need to know the server and client's
+    # ip address, so we add a known ip to NETDEV.
+    my $netdev = get_var('NETDEV', 'eth0');
+    assert_script_run("ip addr add $client_ip/24 dev $netdev") if (is_s390x);
+
+    mutex_wait('IPSEC_SERVER_READY');
+
+    assert_script_run("cd $audit_test::test_dir/ipsec_configuration/toe");
+
+    # Setup the ipip tunnel to the IPSec gateway and test it
```

Comment:
ipip ? a typo?

Reply:
not a typo.
The command is `ip tunnel add tun0 mode ipip remote xxx local xxx`.
So it really 'ipip'

Reply:
got it